pcdscalc/xray.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import periodictable
+import xraylib
+
+import pcdscalc.constants as cst
+
+from .common import get_energy
+from .constants import const, units
+
+
+def molecular_mass(material_id):
+    """Returns the molecular mass of a chemical formula in g"""
+    material_id = cst.chemical_name_to_formula.get(material_id, material_id)
+    form = periodictable.formula(material_id)
+    mass = 0
+    for elem, atoms in form.atoms.items():
+        element_id = elem.symbol
+        mass += cst.atomic_mass[element_id]*atoms
+    return mass
+
+
+def CS_Photo(material_id, energy=None):
```

Comment:
This is atypical naming for a function, it doesn't match pep8
Is this kept the same as old Python, maybe? Or some science/math convention being followed?

Reply:
Yeah this isn't too critical, I'll follow pep8 here instead